galaxy_ng/app/api/ui/views/sync.py

```diff
@@ -0,0 +1,62 @@
+from galaxy_ng.app.constants import COMMUNITY_DOMAINS
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.exceptions import ValidationError
+
+from django.shortcuts import get_object_or_404
+from django.utils.translation import gettext_lazy as _
+
+from pulp_ansible.app import models as pulp_models
+from pulp_ansible.app.tasks.collections import sync as collection_sync
```

Comment:
Here the sync probably comes from `pulp_container` or is a function that we need to create.

Therre is no collection involved with this I guess.

Reply:
@newswangerd Do you know where the sync comes from? I tried `from pulp_container import sync as container_sync` but I don't think it was found

Reply:
https://github.com/pulp/pulp_container/blob/master/pulp_container/app/tasks/synchronize.py#L22 @ShaiahWren 

```
from pulp_container.app.tasks.synchronize import synchronize
```